### PR TITLE
Changes to do interactive plotting.

### PR DESCRIPTION
**Pull request by kadupitiya**
Changes to do interactive plotting.

### CODE REVIEW

**Comment by kadupitiya** [comment]
@jadhao: Please have a look.

File: src/functions.cpp [hunk 1]
```diff
@@ -269,6 +273,33 @@ void compute_density_profile(int cpmdstep, double density_profile_samples, vecto
     return;
 }
 
+void cout_enery_data() {
```
Comment (jadhao) [inline src/functions.cpp]: @kadupitiya energy spelling check please

File: src/functions.h [hunk 2]
```diff
@@ -65,6 +65,8 @@ double compute_MD_trust_factor_R_v(int);
 // display progress bar (code from the internet)
 void progressBar(double);
 
+void cout_enery_data();
```
Comment (jadhao) [inline src/functions.h]: @kadupitiya energy

**Comment by jadhao** [comment]
@kadupitiya approving and merging but please see comments.

**Comment by kadupitiya** [comment]
@jadhao: My bad I have missed the g in energy.
I will correct that in the next version.
I have already called these new functions only under the verbose parameter which is set to false in nanohub specific run.
Thank you.

**Pull request merged**